grafana/Dockerfile

```diff
@@ -1,29 +1,33 @@
-FROM tutum/nginx
-MAINTAINER Vishnu Kannan <vishnuk@google.com>
+FROM gliderlabs/alpine:3.1
+MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+ENTRYPOINT ["/kuisp"]
+CMD ["-p", "8080", "-c", "/opt/grafana/config.js.tmpl=/opt/grafana/config.js", "-s", "/db/=${INFLUXDB_PROTO}://${INFLUXDB_HOST}:${INFLUXDB_PORT}/db/"]
+EXPOSE 8080
 
-ENV GRAFANA_VERSION 1.9.1
-# Environment variables for HTTP AUTH
-ENV HTTP_USER admin
-ENV HTTP_PASS **Random**
+ENV KUISP_VERSION 0.3
 
+ENV GRAFANA_VERSION 1.9.1
+ENV INFLUXDB_NAME k8s
+ENV GRAFANA_DB_NAME grafana
+ENV GRAFANA_DEFAULT_DASHBOARD /dashboard/file/kubernetes.json
 ENV INFLUXDB_PROTO http
-ENV INFLUXDB_HOST localhost
+ENV INFLUXDB_HOST influxdb
```

Comment:
So I see two separate config requirements here.
1. How to access Influxdb internally from Grafana container.
2. How to access Influxdb externally from outside the cluster (which is that of reaching back to the proxy endpoint in the Grafana container).
IIUC, both of those configs are conflated now.